### ISSUE TITLE
Optimize Scripts

### ISSUE THREAD

**Issue by PWRxPSYCHO** [ScriptTactics/ATAK-CoT-Converters#2]
Currently the scripts need to read in a .KML file with the traffic camera data, however it might be possible to utilize a REST API call to fetch the data and then just parse that into the ATAK CoT files.

ex:
```json
{
"id":"4c3de40f-6f13-491f-9253-1fc93cada807",
"name":"Drumgoole Rd W @ Richmond Ave - quad - ptz - 0.162",
"latitude":40.56327,
"longitude":-74.16978,
"area":"Staten Island",
"isOnline":"true",
"imageUrl":"https://webcams.nyctmc.org/api/cameras/4c3de40f-6f13-491f-9253-1fc93cada807/image"
}
```

or

```json
{
    "description": "I-270 & Old Hundred Rd (MD 109)(CAM 165)",
    "routeNumber": 270,
    "milePost": 22.27,
    "lat": 39.2773,
    "lon": -77.3236,
    "commMode": "ONLINE",
    "cctvIp": "strmr5.sha.maryland.gov",
    "routePrefix": "IS",
    "routeSuffix": "",
    "cameraCategories": [
        "Wash. DC"
    ],
    "publicVideoURL": "https://chart.maryland.gov/Video/GetVideo/7a00a1dc01250075004d823633235daa",
    "opStatus": "OK",
    "lastCachedDataUpdateTime": 1703443944276,
    "name": "I-270 & Old Hundred Rd (MD 109)",
    "id": "7a00a1dc01250075004d823633235daa"
}
```

The only fields required for ATAK are:
- Name/Description
- Lat
- Lon
- Area/cameraCategories
- imageUrl/publicVideoURL

Since Python can just convert the JSON to a dict it'll be easier to read through and won't require file I/O

**Comment by PWRxPSYCHO** [comment]
It looks like the publicVideoURL is not compatible with ATAK. The .kml contains the correct URL. This optimization path might not be worth it.

**Comment by PWRxPSYCHO** [comment]
solved with pr #3
